src/System.Diagnostics.Process/pkg/System.Diagnostics.Process.pkgproj

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Diagnostics.Process.csproj">
+      <SupportedFramework>net461;dnxcore50</SupportedFramework>
+    </ProjectReference>
+    <ProjectReference Include="..\ref\4.0\System.Diagnostics.Process.csproj" >
+      <SupportedFramework>net46</SupportedFramework>
+    </ProjectReference>
+    <ProjectReference Include="..\src\facade\4.0.0.0\System.Diagnostics.Process.csproj">
```

Comment:
In keeping with @weshaggard's naming conventions can we use 'net46' here instead?  Also consider combining the projects and just using conditions on TargetGroup.